wiki/How-ALVR-works.md

```diff
@@ -21,11 +21,11 @@ This document was last updated on June 27th 2023 and refers to the master branch
   * Driver communication
   * Driver lifecycle
 * The streaming pipeline: Overview
-* Client-driver communication
+* Device-driver communication
```

Comment:
Same for this document, I don't think we have to rename. I might review the document and update it all at once at a later date.

Reply:
Yea but I've already done it. The question now is whether to keep it renamed

Reply:
I prefer if you undo these changes